docs/faq.md

```diff
@@ -481,4 +482,8 @@ TableStatusRecovery.main(args) --> args is of length two: 1. Database Name 2. Ta
 TableStatus Recovery tool cannot recover table status version files for the below two scenarios
 1. After compaction, if table status file is lost, cannot recover compacted commit transaction, as the lost version file only has merged load details.
 2. After Delete segment by Id/Date, if table status file is lost, cannot recover deleted segment commit transaction, as the lost version file only has the segment status as deleted.
-3. Table status recovery on materialized view table is not supported.
\ No newline at end of file
+3. Table status recovery on materialized view table is not supported.
+
+## Why deleted partition data still showing in file system
+Please enable the feature **moving deleted partition data to trash** by making the `carbon.enable.partitiondata.trash` property true.
```

Comment:
Can mention like below.
By default, the dropped partition data will not be physically removed from the table store until the table is dropped. Enable [carbon.enable.partitiondata.trash](./configuration-parameters.md#system-configuration) property to move all the dropped partitions data to trash during alter table DROP PARTITION operation itself.